src/components/header/Header.vue

```diff
@@ -143,6 +143,12 @@ export default defineComponent({
 
 .m-header {
   background: #fff !important;
+  height: 64px !important;
```

Comment:
could we adjust this styling on the ui library instead of overwritten css?

Reply:
Ok. How about fixing it in another PR?

Reply:
Sure